### ISSUE TITLE
[Toolkits] Code examples should be focusable

### ISSUE THREAD

**Issue by microbit-robert** [microbit-foundation/python-editor-next#700]
Code examples in the toolkits should be focusable to allow a user to copy the code (or perhaps use `enter` to insert the code).
